test/model/store_test.dart

```diff
@@ -1191,6 +1194,21 @@ void main() {
         checkLastRequestFcm(token: '456def');
       }
     }));
+
+    test('use provided appId from packageInfo', () => awaitFakeAsync((async) async {
```

Comment:
nit: a camel-case term like "appId" isn't something that ever appears in normal English capitalization, so it should only be used when it's a proper name — when it's the exact name of something in particular, i.e. usually some identifier in code.

There isn't any identifier `appId` in the code related to this, so that spelling shouldn't appear here either.

Instead, in a plain-English context like a test description, use normal English spacing and capitalization, like "app ID":

```suggestion
    test('use provided app ID from packageInfo', () => awaitFakeAsync((async) async {
```

Reply:
Separately, the behavior this is testing is specific to iOS, so let's have the test description say that:
```suggestion
    test('on iOS, use provided app ID from packageInfo', () => awaitFakeAsync((async) async {
```